ci-operator/step-registry/openshift-qe/perfscale-ibm-data-path-sg/openshift-qe-perfscale-ibm-data-path-sg-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+# Configure ibmcloud
+# to find way to store region and ic_api_key
+export IBMCLOUD_CLI=ibmcloud
+export IBMCLOUD_HOME=/output
+region="${LEASED_RESOURCE}"
+export region
+"${IBMCLOUD_CLI}" config --check-version=false
+echo "logging in..."
+"${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+"${IBMCLOUD_CLI}" plugin list
+resource_group_name=$(jq -r .ibmcloud.resourceGroupName ${SHARED_DIR}/metadata.json)
+ibmcloud target -g $resource_group_name -r $region
```

Comment:
ibmcloud resource group $resource_group_name || exit 1

Reply:
s/ibmcloud/${IBMCLOUD_CLI}/g

Please replace references of `ibmcloud` with ${IBMCLOUD_CLI}